etcdutl/snapshot/v3_snapshot.go

```diff
@@ -49,14 +49,14 @@ import (
 
 // Manager defines snapshot methods.
 type Manager interface {
-	// Save fetches snapshot from remote etcd server and saves data
-	// to target path. If the context "ctx" is canceled or timed out,
+	// Save fetches snapshot from remote etcd server, saves data
+	// to target path and returns server version. If the context "ctx" is canceled or timed out,
 	// snapshot save stream will error out (e.g. context.Canceled,
 	// context.DeadlineExceeded). Make sure to specify only one endpoint
 	// in client configuration. Snapshot API must be requested to a
 	// selected node, and saved snapshot is the point-in-time state of
 	// the selected node.
-	Save(ctx context.Context, cfg clientv3.Config, dbPath string) error
+	Save(ctx context.Context, cfg clientv3.Config, dbPath string) (version string, err error)
```

Comment:
This we could modify as its not exposed via client?

Should we still mention it in the changelog, along with the deprecated `Save` function?

Reply:
This function is not part of client library which has backward compatibility requirements. This is internal function of the `etcdutl` which is the command line program.